### ISSUE TITLE
Error with adding email contact

### ISSUE THREAD

**Issue by nlamirault** [louy/terraform-provider-uptimerobot#15]
I try to add my email as a contact using the provider (It works with the UI). I've got this error : 

```
Error: Error applying plan:
1 error(s) occurred:
* uptimerobot_alert_contact.nicolas_lamirault_email: 1 error(s) occurred:
* uptimerobot_alert_contact.nicolas_lamirault_email: Got error from UptimeRobot: {"message":"You can not add your e-mail as alert contact.","parameter_name":"value","passed_value":"nicolas.lamirault@gmail.com","type":"invalid_parameter"}
```

Any idea ?

**Comment by aequitas** [comment]
I got the same problem. And also a similar problem with the monitor resource type ping:
```
* uptimerobot_monitor.ping: Got error from UptimeRobot: {"parameter_name":"url","passed_value":"basisbeveiliging.nl","type":"invalid_parameter"}
```

**Comment by aequitas** [comment]
My bad, the ping resource was my not specifying the right type for the monitor (http instead of ping).

**Comment by aequitas** [comment]
The account email address is added by default as a contact. So my assumption is the API disallows to add duplicate. I don't think this is an issue with this plugin.

**Comment by louy** [comment]
Good shout @aequitas. I'll see if I get a chance to implement that but feel free to contribute

**Comment by leeif** [comment]
I have the same problem when I want to use the default account email as an alert contact.
Here is a solution I consider:

- Writing an alert contact resource of default account email 

- Importing it into terraform state file
```
terraform import <resource> <id>
```
However the id of the default account email seems like 0xxxx, there is a need to change the type of id to string in the alert contact resource.
https://github.com/louy/terraform-provider-uptimerobot/blob/master/uptimerobot/api/monitor.go#L122

**Comment by aequitas** [comment]
I got some work on this already not finished yet though. I'll try to wrap it up and get a pr out beginning of next week.

**Comment by louy** [comment]
I think adding a data source for alert contacts would be a nice and clean approach to solving this, but I need to verify that it’s doable using the api

I’ve not seen the 0xxxx ids but I don’t see a reason why ids shouldn’t be changed to strings (Ints are probably bad anyway) so feel free to send a PR

**Comment by aequitas** [comment]
https://github.com/louy/terraform-provider-uptimerobot/pull/21

**Comment by louy** [comment]
Fixed in #21
